roles/ovirt-host-deploy-libvirt-guests/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Missing the default values for your variables here?

Reply:
As already mentioned.